matchbox_demo/src/main.rs

```diff
@@ -21,6 +21,9 @@ enum AppState {
 
 const SKY_COLOR: Color = Color::rgb(0.69, 0.69, 0.69);
 
+#[derive(Default, Resource)]
+struct SocketResource(Option<WebRtcSocket>);
```

Comment:
**question** should this be an explicit field like `inner` rather than a tuple?

Reply:
Maybe. When I ported extreme_bevy, I think I had something like this:

```rust
struct RollbackSession {
    socket: Option<WebRtcSocket>,
}
```

Reply:
Or maybe it was match. It's a bit unfortunate that it's so similar to the other Session type :/